spec/features/admin_claim_tasks_update_with_dqt_api_spec.rb

```diff
@@ -102,147 +101,1323 @@ def task_outcome
+              )
+            end
+          end
+        end
+
+        context "with admin claims tasks qualifications passed" do
+          let(:claim) do
+            claimant_submits_claim(
+              claim_attributes: {
+                academic_year: AcademicYear.new(2021),
+                date_of_birth: Date.new(1990, 8, 23),
+                first_name: "Fred",
+                national_insurance_number: "QQ100000C",
+                policy: policy,
+                reference: "AB123456",
+                surname: "ELIGIBLE",
+                tasks: [build(:task, name: "qualifications")],
+                teacher_reference_number: "1234567"
+              }
+            )
+          end
+
+          context "admin claim tasks qualifications view" do
+            before { visit admin_claim_task_path(claim, :qualifications) }
+
+            scenario "shows task outcome previously performed by user" do
+              expect(task_outcome).to have_text("This task was performed by #{claim.tasks.where(name: :qualifications).first.created_by.full_name} on #{I18n.l(claim.tasks.where(name: :qualifications).first.created_at)}")
+            end
+          end
+
+          context "admin claim notes view" do
+            before { visit admin_claim_notes_path(claim) }
+
+            scenario "doesn't show not eligible by automated check" do
+              expect(notes).not_to include(
+                have_text(%r{[Nn]ot eligible}).and(
+                  have_text("by an automated check")
+                )
+              )
+            end
+          end
+        end
+      end
+
+      context "without eligible qualifications" do
+        let(:data) do
+          {
+            qts_award_date: Date.new(
+              claim.eligibility.first_eligible_itt_academic_year.start_year - 1.year,
+              9,
+              1
+            ),
+            itt_subject_codes: ["NoCode"]
+          }
+        end
+
+        context "admin claim tasks view" do
+          before { visit admin_claim_tasks_path(claim) }
+
+          scenario "shows qualifications incomplete" do
+            expect(task("Qualifications")).to have_text("No match")
+          end
+        end
+
+        context "admin claim tasks qualifications view" do
+          before { visit admin_claim_task_path(claim, :qualifications) }
+
+          scenario "shows qualifications question" do
+            expect(page).to have_content("Does the claimant’s initial teacher training (ITT) start/end year and subject match the above information from their claim?")
+          end
+        end
+
+        context "admin claim notes view" do
+          before { visit admin_claim_notes_path(claim) }
+
+          scenario "shows not eligible by an automated check" do
+            expect(notes).to include(
+              have_text("Not eligible").and(
+                have_text("by an automated check")
+              )
+            )
+          end
+        end
+      end
     end
+  end
 
```

Comment:
@nomasprime This feels like far to many levels of nested contexts. There are 6 levels of depth, 5 if you remove what I have asked might be an error. That to me feels too deep.

Have you considered maybe using an alias for `describe/context` as per the documentation?
https://relishapp.com/rspec/rspec-core/v/3-8/docs/example-groups/aliasing

Each of the `view` type contexts feels like it should be a `describe`

Maybe this would work

```ruby
RSpec.feature "Admin claim tasks update with DQT API"
  describe "EarlyCareerPayments policy" #1
    describe "with submitted claim" #2  (or maybe even join 1 + 2 to a single describe)
    # There is no unsubmitted claim so why do we need the `with submitted claim`?

      context "with matching DQT identity" #3
        describe "admin claim tasks view" #4
        describe "admin claim tasks identity confirmation view" #4
        describe "admin claim notes view" #4
        context "except national insurance number" #4
          describe "admin claim tasks view" #5
          describe "admin claim tasks identity confirmation view" #5
          describe "admin claim notes view" #5
        context "except matching teacher reference number" #4
          describe "admin claim tasks view" #5
          describe "admin claim tasks identity confirmation view" #5
          describe "admin claim notes view" #5
        context "except matching first name" #4
          describe "admin claim tasks view" #5
          describe "admin claim tasks identity confirmation view" #5
          describe "admin claim notes view" #5
        context "except matching surname" #4
          describe "admin claim tasks view" #5
          describe "admin claim tasks identity confirmation view" #5
          describe "admin claim notes view" #5
            context "admin claim notes view" #6
        context "with middle names" #4
          describe "admin claim tasks view" #5
          describe "admin claim tasks identity confirmation view" #5
          describe "admin claim notes view" #5
        context "except matching date of birth" #4
          describe "admin claim tasks view" #5
          describe "admin claim tasks identity confirmation view" #5
          describe "admin claim notes view" #5
        context "with admin claims tasks identity confirmation passed" #4
          describe "admin claim tasks identity confirmation view" #5
          describe "admin claim notes view" #5
        context "with teacher status alert" #4
          describe "admin claim tasks view" #5
          describe "admin claim tasks identity confirmation view" #5
          describe "admin claim notes view" #5
          describe "admin claim" #5

      context "without matching DQT identity" #3
        describe "admin claim tasks view" #4
        describe "admin claim tasks identity confirmation view" #4
        describe "admin claim notes view" #4

      context "with eligible qualifications" #3
        describe "admin claim tasks view" #4
        describe "admin claim tasks qualifications view" #4
        describe "admin claim notes view" #4
        context "except QTS award date" #4
          describe "admin claim tasks view" #5
          describe "admin claim tasks qualifications view" #5
          describe "admin claim notes view" #5
        context "except ITT subjects codes" #4
          describe "admin claim tasks view" #5
          describe "admin claim tasks qualifications view" #5
          describe "admin claim notes view" #5
        context "with admin claims tasks qualifications passed" #4
          describe "admin claim tasks qualifications view" #5
          describe "admin claim notes view" #5

      context "without eligible qualifications" #3
        describe "admin claim tasks view" #4
        describe "admin claim tasks qualifications view" #4
        describe "admin claim notes view" #4
```